#! /bin/sh /usr/share/dpatch/dpatch-run
## 025_define_GNU_SOURCE_earlier.dpatch by Stefan Fritsch <sf@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: _GNU_SOURCE needs to be #defined before #including stuff
## DP: (causing FTBFS on hurd, #530286)

@DPATCH@
diff -urNad trunk~/include/apr.h.in trunk/include/apr.h.in
--- trunk~/include/apr.h.in	2009-06-07 19:10:19.000000000 +0200
+++ trunk/include/apr.h.in	2009-06-07 20:50:52.261954142 +0200
@@ -33,6 +33,10 @@
  * for Win32 or Netware by those build environments, respectively.
  */
 
+#ifndef _GNU_SOURCE
+# define _GNU_SOURCE 1
+#endif
+
 /**
  * @defgroup APR Apache Portability Runtime library
  * @{
